.github/workflows/ci.yml

```
@@ -0,0 +1,8 @@
+name: Playground Push Action
+on: [push]
```

It's often valuable to test on PRs as well
```suggestion
on: [push, pull_request]
```

---

Yes, absolutely.  I was just trying to confirm permissions with this initial commit. Once I get the permissions sorted out there will be many changes.